zebra-chain/src/sapling/output.rs

```diff
@@ -134,12 +134,45 @@ impl ZcashSerialize for OutputInTransactionV4 {
 
 impl ZcashDeserialize for OutputInTransactionV4 {
     fn zcash_deserialize<R: io::Read>(mut reader: R) -> Result<Self, SerializationError> {
+        // # Consensus
+        //
+        // > Elements of a Output description MUST be valid encodings of the types given above.
```

Comment:
I would add something like:
```suggestion
        // > Elements of an Output description MUST be valid encodings of the types given above.
        //
        // Note that _above_ refers to a list of types placed above the consensus rule in the specification.                                                
        // These types are listed below.
```
However, I noticed that this and similar rules are mentioned multiple times in this PR, so I'm not sure if we should add such a note to each occurrence.

Reply:
I see your point. Since the the rule is a quote, I assumed it would be clear that "above" is in the spec, but I agree it can be confusing. On the other hand we'd have to change a lot of comments, and I'd prefer to not do that right now. Is that OK?

Reply:
Totally.